src/main/kotlin/org/stellar/walletsdk/utils/Operation.kt

```diff
@@ -0,0 +1,26 @@
+package org.stellar.walletsdk.utils
+
+import org.stellar.sdk.BeginSponsoringFutureReservesOperation
+import org.stellar.sdk.EndSponsoringFutureReservesOperation
+import org.stellar.sdk.Operation
+
+class Operation {
+  companion object {
+    fun createSponsoredOperation(
+      sponsorAddress: String,
+      accountAddress: String,
+      operation: Operation
+    ): List<Operation> {
+      return listOfNotNull(
+        // Start reserve sponsoring
+        BeginSponsoringFutureReservesOperation.Builder(accountAddress)
+          .setSourceAccount(sponsorAddress)
+          .build(),
+        // Operation to sponsor
+        operation,
+        // End reserve sponsoring
+        EndSponsoringFutureReservesOperation(accountAddress)
+      )
+    }
+  }
+}
```

Comment:
In Kotlin, you may define functions in a `.kt` file. 

This is an [example](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/integration-tests/src/main/kotlin/org/stellar/anchor/platform/TestHelper.kt). 

```suggestion
    fun createSponsoredOperation(
      sponsorAddress: String,
      accountAddress: String,
      operation: Operation
    ): List<Operation> {
      return listOfNotNull(
        // Start reserve sponsoring
        BeginSponsoringFutureReservesOperation.Builder(accountAddress)
          .setSourceAccount(sponsorAddress)
          .build(),
        // Operation to sponsor
        operation,
        // End reserve sponsoring
        EndSponsoringFutureReservesOperation(accountAddress)
      )
    }

```

Reply:
I'm so glad to see that not everything needs to go in class in Kotlin. :sweat_smile: I'll move helper methods directly to the `utils` folder. :+1: